openedx/features/smart_referral/tasks.py

```diff
@@ -15,3 +17,14 @@ def task_send_referral_and_toolkit_emails(contact_emails, user_email):
         })
 
     mandrill_client.send_mail(MandrillClient.REFERRAL_SOCIAL_IMPACT_TOOLKIT, user_email, context={})
+
+
+@task(routing_key=settings.HIGH_PRIORITY_QUEUE)
+def task_send_referral_follow_up_email(contact_email_list):
+    """Send follow up referral email to contact email."""
+    for contact_email in contact_email_list:
+        status = MandrillClient().send_mail(MandrillClient.REFERRAL_FOLLOW_UP_EMAIL, contact_email, context={
+            'root_url': settings.LMS_ROOT_URL,
+        })
+        if status[0]['status'] == 'sent':
+            SmartReferral.objects.filter(contact_email=status[0]['email']).update(is_referral_step_complete=True)
```

Comment:
Can we write `contact_email=contact_email` instead of `contact_email=status[0]['email']`

Reply:
Yes, both are same but I choose to get email address from status of email to avoid uncertainty as it runs as asynchronous so it may be a little possibility that we would get different email address.